### ISSUE TITLE
Data fetch() from zKillboard failed

### ISSUE THREAD

**Issue by barndawgie** [goryn-clade/pathfinder#40]
Our maps on the public pathfinder (at https://www.pathfinder-w.space) have started failing to pull from zKillboard and displaying an error in the bottom corner:
![zKillFetchFailure](https://user-images.githubusercontent.com/11252867/137566492-af89a1ef-29a8-4264-84c1-3c92c9eb98f5.png)
.

**Comment by BrandXX** [comment]
Same issue on my servers. I have not had a chance to dig in.

**Comment by ppfeufer** [comment]
Might have something to do with this:

- https://github.com/zKillboard/zKillboard/wiki/API-(Killmails)
- https://www.reddit.com/r/Eve/comments/q8bs4b/zkillboard_all_filters_disabled_with_exception_of/

**Comment by samoneilll** [comment]
Yeah, PF makes calls with the "no-items" and "no-attacker" filters. I'm not entirely sure what they did, there's no info about it in the API docs other than to say that they're not deprecated. 

I believe we can simply update the API call to `api/kills/solarSystemID/{systemId}/pastSeconds/86400/`. I'll try to have it in the next release with the SSO updgrades.

**Comment by lunedis** [comment]
> Yeah, PF makes calls with the "no-items" and "no-attacker" filters. I'm not entirely sure what they did, there's no info about it in the API docs other than to say that they're not deprecated.

`no-items` makes it so zKill doesn't return the fitted and dropped items for every kill in the return set.
`no-attackers` is the same but with all the attackers on the killmail.

Both are mainly used to reduce the amount of data returned.

Looking at the error message the problem is probably the combination of `/solarSystemId` and `/npc/0` (without having tested it myself).

**Comment by samoneilll** [comment]
The error is because the no-items endpoint no longer returns json, but instead a text response to say the endpoint is deprecated. https://github.com/zKillboard/zKillboard/wiki/API-(Killmails)#deprecated

You can try it yourself: 

```sh
$ curl https://zkillboard.com/api/no-items/w-space/no-attackers/npc/0/solarSystemID/31002565/pastSeconds/86400/
> "no-items has been permanently disabled."

**Comment by samoneilll** [comment]
https://github.com/goryn-clade/pathfinder/commit/1966c3d0889fec9dc5572e82deebfc404068d04d 

Updating the fetch-string to drop the no-items and no-attackers modifiers appears to fix the issue. I will include v2.1.3 release.

**Comment by BrandXX** [comment]
Thank you

**Comment by samoneilll** [comment]
Solved in release 2.1.3
